2015/proposals/akshay.md

```diff
@@ -0,0 +1,107 @@
+# Title
+
+Add taxonomic name resolution to the EcoData Retriever to facilitate data 
+science approaches to ecology.
+
+## Abstract
+
+The EcoData Retriever is an engine which automates the tasks of finding,
+downloading, and cleaning up ecological data files, and then stores them in a
+local database of your choice.The program cleans and munges the datasets and
+formats them  before inserting it into your database.
+
+This project will tackle the problem of resolving scientific names stored 
+inconsistently across different datasets. This would allow the users to combine
+different datasets and do interesting things.
```

Comment:
I agree with @r-gaia-cs. Even something as simple as replacing `and do interesting things` with `to scientific questions that can't be answered using a single datasets alone` would be an improvement.